Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -193,7 +193,7 @@ public void onDismiss(DialogInterface dialog) {
                 .show();
     }
 
-    private void removePaymentMethodNonce(PaymentMethodNonce paymentMethodNonceToDelete) {
+    void removePaymentMethodNonce(PaymentMethodNonce paymentMethodNonceToDelete) {
```

Comment:
random q: could this still be private?

Reply:
Yes -  I made it package-private to make testing easier. This method gets called from the confirmation click in the cancel alert that we build in `deleteVaultedPaymentMethod`. Seemed more straightforward to test this method directly than mock the alert dialog but could investigate that if we'd rather keep this private.

Reply:
Ah nice yeah. That makes sense. We could throw in an `@VisibleForTesting` annotation so a lint error occurs if someone tries to call it externally.